.github/workflows/create_containers.yml

```
@@ -1,11 +1,40 @@
+  #
+  # The Arch container is force-rebuilt on schedule so we always run against the
+  # latest one.
+  ARCH_CONTAINER_TAG: "latest"
+  CENTOS_CONTAINER_TAG: "2026-09-03.1"
+  DEBIAN_CONTAINER_TAG: "2026-09-03.1"
+  UBUNTU_CONTAINER_TAG: "2026-09-03.1"
+  FEDORA_CONTAINER_TAG: "2026-09-03.1"
 
 jobs:
   push_to_registry:
+    if: ${{ github.event_name == 'push' || !inputs.distro_filter || matrix.distros.DISTRO == inputs.distro_filter }}
```

actionlint (1.7.12)

[error] context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)